packages/web-react/transforms/transformValueExample.ts

```diff
@@ -0,0 +1,67 @@
+// Install jscodeshift if you haven't already
+// npm install -g jscodeshift
+
+// Run with:
+// jscodeshift --transform ./transforms/transformValueExample.ts --extensions tsx ./transforms/src
+
+// Result:
+// Will replace *-left to *-start etc. Tooltip components imported from '@lmc-eu/spirit-web-react'
+
+const transform = (fileInfo, api) => {
+  const j = api.jscodeshift;
+  const root = j(fileInfo.source);
+
+  // Find import statements for the specific module
+  const importStatements = root.find(j.ImportDeclaration, {
```

Comment:
how does this work? The adopters might use:
* @lmc-eu/spirit-web-react
* @lmc-eu/spirit-web-react/components
* @lmc-eu/spirit-web-react/components/Tooltip

Does all of these work?